digiroad2-geo/src/main/scala/fi/liikennevirasto/digiroad2/lane/LanePartitioner.scala

```diff
@@ -28,28 +21,159 @@ object LanePartitioner extends GraphPartitioner {
     (mainLanes, additionalLanes)
   }
 
-  def partitionBySideCodeAndLaneCode(linearAssets: Seq[PieceWiseLane]): Seq[Seq[PieceWiseLane]] = {
-    val (lanesOnOneDirectionRoad, lanesOnTwoDirectionRoad) = linearAssets.partition(_.sideCode == SideCode.BothDirections.value)
-    val (lanesOnTowards, lanesOnAgainst) = lanesOnTwoDirectionRoad.partition(_.sideCode == SideCode.TowardsDigitizing.value)
+  def partitionBySideCodeAndLaneCode(linearAssets: Seq[PieceWiseLane], roadLinks: Map[Long, RoadLink]): Seq[Seq[PieceWiseLane]] = {
```

Comment:
miksi paluu tyyppi on kaksi sisäkkäistä seq ?, vaikuttaa palauttavan vain yhden listan

Reply:
palauttaa yhden listan, jonka sisällä listoihin jaettuna kaistat. Tämä sen takia, että kaikkia kaistoja ei voida käydä kerralla läpi geotoolsin GraphPartitionerin ongelman takia.

Reply:
ok varmaan pitää sitten mennä näin